x-pack/plugins/observability_solution/observability/server/services/slo/find_slo_groups.ts

```diff
@@ -75,7 +77,7 @@ export class FindSLOGroups {
         aggs: {
           groupBy: {
             terms: {
-              field: groupBy,
+              field: groupBy === 'remoteCluster' ? '_index' : groupBy,
```

Comment:
See my note on this group by in the public files, to avoid having this custom logic.